blocksync/buffer.go

```diff
@@ -71,7 +71,7 @@ func (b *blockBuffer) AddBlock(tipHeight uint64, blk *peerBlock) (bool, uint64)
 	defer b.mu.Unlock()
 	blkHeight := blk.block.Height()
 	if blkHeight <= tipHeight {
-		return false, 0
+		return false, blkHeight
```

Comment:
no impact to current logic

Reply:
yep. The change is helpful to log the change of target height by the block of higher height.